base/data/src/main/java/org/openscience/cdk/DefaultChemObjectBuilder.java

```diff
@@ -117,99 +117,136 @@ private static boolean getSystemProp(final String key, final boolean defaultValu
 
-        // chem object sets
-        factory.register(IAtomContainerSet.class, AtomContainerSet.class);
-        factory.register(IMolecularFormulaSet.class, MolecularFormulaSet.class);
-        factory.register(IReactionSet.class, ReactionSet.class);
-        factory.register(IRingSet.class, RingSet.class);
-        factory.register(IChemModel.class, ChemModel.class);
-        factory.register(IChemFile.class, ChemFile.class);
-        factory.register(IChemSequence.class, ChemSequence.class);
-        factory.register(ISubstance.class, Substance.class);
+                        @Override
+                        public void modify(DoubleBondStereochemistry instance) {
+                            instance.setBuilder(self);
+                        }
+                    });
 
-        // stereo components (requires some modification after instantiation)
-        factory.register(ITetrahedralChirality.class, TetrahedralChirality.class,
-                new DynamicFactory.CreationModifier<TetrahedralChirality>() {
+            // miscellaneous
+            factory.register(IMapping.class, Mapping.class);
+            factory.register(IChemObject.class, ChemObject.class);
+        }
+
+        /**
+         * {@inheritDoc}
+         */
+        @Override
+        public <T extends ICDKObject> T newInstance(Class<T> clazz, Object... params) throws IllegalArgumentException {
+            return factory.ofClass(clazz, params);
+        }
 
-                    @Override
-                    public void modify(TetrahedralChirality instance) {
-                        instance.setBuilder(self);
-                    }
-                });
-        factory.register(IDoubleBondStereochemistry.class, DoubleBondStereochemistry.class,
-                new DynamicFactory.CreationModifier<DoubleBondStereochemistry>() {
+        /**
+         * {@inheritDoc}
+         */
+        @Override
+        public IAtom newAtom() {
+            return new Atom();
+        }
 
-                    @Override
-                    public void modify(DoubleBondStereochemistry instance) {
-                        instance.setBuilder(self);
-                    }
-                });
+        /**
+         * {@inheritDoc}
+         */
+        @Override
+        public IBond newBond() {
+            return new Bond();
+        }
 
-        // miscellaneous
-        factory.register(IMapping.class, Mapping.class);
-        factory.register(IChemObject.class, ChemObject.class);
+        /**
+         * {@inheritDoc}
+         */
+        @Override
+        public IAtomContainer newAtomContainer() {
+            if (CDK_LEGACY_AC)
+                return new AtomContainer(0, 0, 0, 0);
+            else
+                return new AtomContainer2(0, 0, 0, 0);
 
+        }
     }
 
     /**
-     * Access the singleton instance of this DefaultChemObjectBuilder. 
+     * Access the singleton instance of this SilentChemObjectBuilder.
```

Comment:
The SilentChemObjectBuilder is now the default? Yeah, the original name was crappy. But this change still is in DefaultChemObjectBuilder, so I'm a bit confused...

Reply:
Nope that was a copy-paste error :-)

Reply:
I've fixed the doc now - thanks for spotting.

Reply:
Okay, this slip shows up in other implementations too... so, if not done yet, plz check the full patch for other such unintended changes.